### PR TITLE
FEATURE: Sticky Note node

### PR DESCRIPTION
**Pull request by 0xi4o**
- Adds a "Sticky Note" node under "Utilities". This node can be used to add notes to a Chatflow to explain what a node or a group of nodes does, explain input and output parameters, etc.

**Update**: Made the node look like a sticky note to visually distinguish it from other nodes. I think this will reduce visual clutter when you have a lot of nodes and help avoid confusion.

Here's how it looks:
Flowise-Sticky Notes

### CODE REVIEW

File: packages/ui/src/views/canvas/LightTooltip.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { styled } from '@mui/material/styles'
```
Comment (HenryHengZJ) [inline packages/ui/src/views/canvas/LightTooltip.js:1]: Do u think it make sense to refractor this into `ui-component/tooltip` and maybe rename as smtg that is specific like `NodeTooltip` as its only used for `CanvasNode` right?

**Comment by HenryHengZJ** [comment]
We need to also update files under `ui/views/marketplaces`. With sticky note, it appears like this:
image

File: packages/ui/src/views/canvas/LightTooltip.js (hunk 1 above)
Comment (0xi4o) [reply packages/ui/src/views/canvas/LightTooltip.js]: I've moved the component to `ui-component/tooltip` and renamed it to `NodeTooltip`.

File: packages/ui/src/views/canvas/CardWrapper.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// material-ui
```
[the remark this answers is not in the document]
Comment (0xi4o) [reply packages/ui/src/views/canvas/CardWrapper.js]: I've moved it to `ui-component/cards` and renamed it to `NodeCardWrapper`.

**Comment by 0xi4o** [comment]
> We need to also update files under `ui/views/marketplaces`. With sticky note, it appears like this: image

Updated the files under `marketplaces` so it renders sticky notes properly :+1:

**Review by HenryHengZJ — APPROVED** [review]
awesome thanks @0xi4o !

**Comment by maximometascript** [comment]
I'm seeing this:
image

Kinda wishing I didn't delete my old information flow in favor of these. I didn't know I was going to brick them by closing the flow.

**Comment by 0xi4o** [comment]
@maximometascript can you open an issue with more details? I have the latest changes and I can't seem to replicate this.

**Pull request merged**